bubble-wrap-style.yaml

```diff
@@ -3154,13 +3275,14 @@ layers:
         draw:
             lines:
                 order: global.feature_order
-#                color: red
-#                width: [[9, 1px], [14, 2px], [16, 3px], [17, 8m]]
+                #debug:
```

Comment:
Is keeping this useful? (Fine if so.)

Reply:
Yes... I keep some debugging in when I keep going back to debug (see buildings and roads).